### ISSUE TITLE
Automatic logout problem (cannot efficiently login)

### ISSUE THREAD

**Issue by Mizumaky** [web-pal/chronos-timetracker#261]
## Expected Behavior
- I log in to the app once, then never again OR the app sends me to the default browser for each login (for autofill to work) OR the app remembers my credentials so that i do not have to fill them again when logged out (and it actually works)

## Actual Behavior
- After a few days, clicking refresh loads an empty list of tasks - that's when i know, i was logged out
- I close the app, the select Login to existing account
- I select my account
- The app says: "Can not authenticate user. Please try again"
- I manually input my team name
- I click Microsoft login
- I manually input my email
- I manually input my password
- I tick the checkbox "Don't show this again" when being asked "Stay signed in?" and click Yes (with no actual effect)
- I get logged in
TLDR That is too many inconvenient steps (and spent unnecessary time) for this tool to be actually useful.

## Steps to Reproduce the Problem

  1. as described above

## Specifications

  - Chronos-timetracker version: 6.3.0
  - OS: Windows
